C/test/Makefile

```diff
@@ -21,7 +21,7 @@ coverage: yahdlc_test
 	@lcov  --remove all.info "yahdlc_test.cpp" "boost/*" "c++/*" -o report.info -q
 
 coveralls: coverage
-	@coveralls-lcov -t $(REPO_TOKEN) report.info
+	coveralls-lcov -t $(REPO_TOKEN) report.info
```

Comment:
Maybe you can just add this for now? Think the build should anyway continue even though the coverage upload failed

```suggestion
	@coveralls-lcov -t $(REPO_TOKEN) report.info || true
```